### PR TITLE
Added a unit test for uploading a file.

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: website/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python3"
+}
```
Comment (aniehuser) [inline website/.vscode/settings.json]: new line

File: website/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+    "python.linting.enabled": true,
+    "python.pythonPath": "/usr/local/bin/python3"
```
Comment (aniehuser) [inline website/.vscode/settings.json]: ~would this still work on a non UNIX device?~

We should probably add a .gitignore to the `website` directory in it of itself with Django and .vscode settings files ignored

File: website/yw_website/script_upload/tests.py [hunk 3]
```diff
@@ -1,3 +1,26 @@
...
+        # Send dummy request to upload view
+        post_test = self.client.post(path='/', data={'document': f})
+        model_form_upload(post_test.wsgi_request)
+
+        # Assert that file was successfully uploaded to our system
+        file_path = os.getcwd() + "/media/documents/" + file_name
+        self.assertTrue(os.path.isfile(file_path))
+
+        # Clean up after test
+        os.remove(file_path)
```
Comment (aniehuser) [inline website/yw_website/script_upload/tests.py:26]: assert that file was removed

**Review by aniehuser — CHANGES_REQUESTED** [review]
What do you think about having a [separate .gitignore](https://www.atlassian.com/git/tutorials/saving-changes/gitignore#shared) files in different root directories in this repository?

File: website/.vscode/settings.json (hunk 1 above)
Comment (clundin55) [reply website/.vscode/settings.json]: This should not be added to the commit as that is my personal vs code linter. I'll look into updating the git ignore.

**Comment by clundin55** [comment]
@aniehuser It seems that vs code files are already in our git ignore. Something seems to be up with our git ignore as it is not working with the compiled python files and the database files.

**Comment by aniehuser** [comment]
LGTM :)

**Review by aniehuser — APPROVED** [review]

**Comment by aniehuser** [comment]
OK now looks good to me :)

**Pull request merged**